src/experimentExtractor/constants.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
```

Comment:
Love this; could we maybe inspect `process.env` for these in case someone wants to override? e.g. `EXPERIMENT_COMPONENT_NAME: process.env.EXPERIMENT_COMPONENT_NAME || 'Experiment',`

Reply:
Frankly, I do not like the idea of overriding w/o the exact case. I can easily set overriding here, but I don't know if it will work in all other parts of the code.

Reply:
Sounds good. Yeah, without a clear idea of acceptable / unacceptable overrides this risks causing more problems than it solves.

Nonetheless, can you please log a tech debt? e.g. "next-experiments' constants cannot be overridden by project users"

Reply:
Done. https://www.notion.so/sheertex/next-experiments-constants-cannot-be-overridden-by-project-users-12262b7f3edb4bacb6a4777e1dee2f49